src/quo2/foundations/colors.cljs

```diff
@@ -183,6 +183,24 @@
       (let [base-color (get-in colors-map [(keyword color) suffix])]
         (if opacity (alpha base-color (/ opacity 100)) base-color))))))
 
+(defn- opacity->hex [opacity]
+  (case opacity
+    0 "00"
+    10 "1A"
+    20 "33"
+    30 "4d"
+    40 "66"
+    50 "80"
+    60 "99"
+    70 "B3"
+    80 "CC"
+    90 "E6"
+    "FF"))
+
+(defn custom-hex-color [color light-opacity dark-opacity]
+  (str color (opacity->hex (if (theme/dark?) dark-opacity light-opacity))))
+
+;; TODO (OmarBasem): custom-color-by-theme supports only string colors, so maybe we can get rid of it.
```

Comment:
Hi @OmarBasem,
Comment is at the wrong place, [suggestion](https://github.com/status-im/status-mobile/pull/14494/files#r1047095653) is about later removing `custom-hex-color` not `custom-color-by-theme`.
Groups and other parts of the app is using colors based on name/string.

I have no idea why even we are using `(colors/custom-hex-color color 50 60)`.
As I [mentioned](https://github.com/status-im/status-mobile/pull/14494/files#r1042148417) above 50 & 60 are not opacity they are just color variants.
And also currently we are getting the group avatar color as hex, but that's only for now.
Either we can just follow our approach of custom-color-by-them and hard-code primary color for now, or if we want to use the hex color provided by the group, we should directly use that. Instead of changing opacity.

Reply:
Alright I understand, but the 50 and 60 in the group avatar are actually opacity, that's how they are in the designs. And the group colors comes as hex, that's why we need this method. Since `custom-color-by-theme` is used by other components, then I guess we need both, right?

Reply:
>  but the 50 and 60 in the group avatar are actually opacity

they are not opacity - https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System?node-id=179%3A9760&t=XVp7uXCCPclgNDXg-0

![1](https://user-images.githubusercontent.com/17097240/207587318-94bfaa08-f328-43c4-98b3-001bd03158b2.png)
![2](https://user-images.githubusercontent.com/17097240/207587332-13a8ceb0-b44d-43ff-ae05-4f3210aaa788.png)
![3](https://user-images.githubusercontent.com/17097240/207587335-23852932-61f1-4e81-8203-3416ccc47b14.png)

Reply:
@Parveshdhull you are right. My bad. Sorry! I will fix it.

Reply:
@Parveshdhull Since the colors come as hex from the backend, and the designs are using specific customizations of the colors, either the designs need to be updated, or the way colors are represented needs to change. So for now I think the best option is to let the `group-avatar` use the color as it is.